tests/cross/src/test/scala/tests/pc/CompletionSuite.scala

```diff
@@ -1623,4 +1623,35 @@ class CompletionSuite extends BaseCompletionSuite {
     topLines = Some(1),
   )
 
+  check(
+    "type-with-params",
+    s"""|object O {
+        | type TTT[A <: Int] = List[A]
+        | val t: TT@@
+        |}
+        |""".stripMargin,
+    "TTT[A] = O.TTT",
+    compat = Map(
+      "3" -> "TTT[A <: scala.Int] >: List[A] <: List[A]"
```

Comment:
The other bounds are on the result. Do we not want it? I guess we could instead of `>: List[A] <: List[A]` write `= List[A]` if the upper and lower bounds are the same.

Reply:
I think this might already be handled in Scala 2, which we could maybe reuse, but we don't want to show bounds if they are exactly the same, that brings no additional information to the user